### PR TITLE
About page finalisation

### PR DESCRIPTION
**Pull request by PettraV**
@fbbunny moved members to members page, therefore I deleted the unnecessary code. I cannot decrease font-size of  class=intro paragraph. Fonts should be 12 px, but they look at least 20px. Any idea how to resolve this before the merge? Now let me continue with querysets....Thank you :)

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -127,28 +127,25 @@ img{
...
-	color: transparent;
-}
 
 /* paragraph*/
-.intro{
+.intro {
 	text-align: justify;
 	color: #5C5C5C;
-	font-size: 20px;
+	font-size: 1px;
```
Comment (gandalfar) [inline static/css/style.css]: Whatever you're trying to do here - this doesn't look right.

**Comment by gandalfar** [comment]
You're mixing migrations and design in this pull request. In addition to that, you're trying to pull in migration without any models.py changes. This won't work.

File: static/css/style.css (hunk 1 above)
Comment (PettraV) [inline static/css/style.css]: you are right I was experimenting and I do not know why this is not working. I cannot change the size of the main content...

Comment (gandalfar) [inline static/css/style.css]: Well, then it doesn't belong into PR but into an Issue :)

**Comment by PettraV** [comment]
i do not know how this happen, I was working on a branch...What should I do...delete the PR?

**Comment by gandalfar** [comment]
No, just remove that migration from this branch.

**Comment by PettraV** [comment]
pls explain :)

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,2 @@
 Django==1.7.1
-wsgiref==0.1.2
```
Comment (goranche) [inline requirements.txt]: please do *not* remove this from the requirements file...
a.k.a. put it back :smile:

Comment (PettraV) [inline requirements.txt]: done

**Comment by goranche** [comment]
now you still need to remove the migration, as @gandalfar mentioned already

    api/migrations/0002_auto_20150201_0824.py

we can't see what changed you have made to your local repository, so please make sure you don't need the migration on the 'about' branch. check if you have made any changed to the model, that require the migration to be applied. if you're sure you've not, just delete the file and add a commit...

**Comment by goranche** [comment]
also, learn about git staging and how not to include all files in a git commit

**Comment by PettraV** [comment]
thx for the hint :) migrations deleted, should i delete also 00001?

**Comment by PettraV** [comment]
As regard the staging I have a problem I do not understand. I use different branches and push on github branches, but even if i work different files on different branches, i see on git status allways all changes and have to be careful what i push on git hub branch and what not. Is this normal? I always thought branches are completely separate environments even with status changes.

File: static/css/style.css [hunk 3]
```diff
@@ -127,28 +127,25 @@ img{
...
-	color: transparent;
-}
 
 /* paragraph*/
-.intro{
+.intro {
 	text-align: justify;
 	color: #5C5C5C;
-	font-size: 20px;
+	font-size: 12px;
```
Comment (livike) [inline static/css/style.css]: You have decreased the font size, but the "jumbotron p" from bootstrap os overwriting that.
You can this clearly see in Web inspector by inspecting you paragraph element.
Btw: do we really need jumbotron here?

Comment (PettraV) [inline static/css/style.css]: I used the web inspection but in CSS there is no "jumbotron p"

Comment (PettraV) [inline static/css/style.css]: I like jubmbotron :)

**Pull request merged**